zuul-core/src/main/java/com/netflix/zuul/netty/server/Server.java

```diff
@@ -354,8 +365,19 @@ private void initializeTransport()
             Executor workerExecutor = new ThreadPerTaskExecutor(workerThreadFactory);
 
             Map<ChannelOption<?>, Object> extraOptions = new HashMap<>();
-            boolean useNio = FORCE_NIO.get();
-            if (!useNio && epollIsAvailable()) {
+            final boolean useNio = FORCE_NIO.get();
+            final boolean useIoUring = FORCE_IO_URING.get();
+            if (useIoUring) {
```

Comment:
Shouldn't this also check if it's available?

Reply:
Fixed.